itou/archive/utils.py

```diff
@@ -37,3 +41,41 @@ def count_related_subquery(model, fk_field, outer_ref_field, extra_filters=None)
     return Coalesce(
         model.objects.filter(**filters).values(fk_field).annotate(count=Count(outer_ref_field)).values("count"), 0
     )
+
+
+def inactive_jobseekers_without_recent_related_objects(inactive_since, batch_size=None, notified=False):
```

Comment:
```suggestion
def inactive_jobseekers_without_recent_related_objects(inactive_since, batch_size=None, *, notified):
```
pour forcer le dev à spécifier `notified`.

Car le fait que `inactive_jobseekers_without_recent_related_objects(une_date)` ne renvoie que des candidats pas encore notifiés n'est pas du tout évident.

Reply:
bien vu